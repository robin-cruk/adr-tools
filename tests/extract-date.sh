adr new Decision
_adr_date 1
