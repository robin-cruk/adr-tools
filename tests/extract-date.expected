adr new Decision
doc/adr/0001-decision.md
_adr_date 1
1992-01-12
